scripts/coder-dev.sh

```diff
@@ -59,7 +60,20 @@ esac
 runcmd=("${CODER_DEV_BIN}")
 if [[ "${DEBUG_DELVE}" == 1 ]]; then
 	set -x
-	runcmd=(dlv debug --headless --continue --listen 127.0.0.1:12345 --accept-multiclient ./cmd/coder --)
+	build_flags=(
+		--os "$GOOS"
+		--arch "$GOARCH"
+		--output "$CODER_DELVE_DEBUG_BIN"
+		--debug
+	)
+	if [[ "$BINARY_TYPE" == "coder-slim" ]]; then
+		build_flags+=(--slim)
+	fi
+	# All the prerequisites should be built above when we refreshed the regular
+	# binary, so we can just build the debug binary here without having to worry
+	# about/use the makefile.
+	./scripts/build_go.sh "${build_flags[@]}"
+	runcmd=(dlv exec --headless --continue --listen 127.0.0.1:12345 --accept-multiclient "$CODER_DELVE_DEBUG_BIN" --)
 fi
 
-CGO_ENABLED=0 exec "${runcmd[@]}" --global-config "${CODER_DEV_DIR}" "$@"
+exec "${runcmd[@]}" --global-config "${CODER_DEV_DIR}" "$@"
```

Comment:
Howcome you dropped the `CGO_ENABLED` setting? We'll need that I'm afraid

Reply:
`build_go.sh` does that I believe

Reply:
`build_go.sh` includes it, it was only added because `dlv debug` runs `go build` so it was necessary to include previously.